pkg/sdk/brn/brn.go

```diff
@@ -29,8 +29,9 @@ const SchemePrefix = Scheme + ":"
 
 // Resource type constants
 const (
-	SecretResourceType  = "secret"
-	ClusterResourceType = "cluster"
+	SecretResourceType  = "secret"  // resource ID: <secret sha id>
+	ClusterResourceType = "cluster" // resource ID: <cluster id>
+	NodeResourceType    = "node"    // resource ID: <cluster id>/<hostname>
```

Comment:
Why not `<cluster id>/<node name>`? I realize the two are usually the same, but not always, are they?

Reply:
pke doesn't (or can't) explicitly know or control the kubernetes node name, we just hope it'll be the (then actual) hostname of the machine

Reply:
This feels like a bad compromise for the sake of PKE. Coming from a Pipeline perspective, nodes are identifiable by their name. PKE not being able to control it can be true, but is there really no way to find the node name out?